renovate.json

```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base"],
+  "extends": [
+    "local>nv-gha-runners/renovate-config",
+    "group:allNonMajor"
+  ],
   "schedule": ["before 4am on the first day of the month"]
```

Comment:
```suggestion
  ]
```

We can remvoe the `schedule` property here since it's included in the shared config